src/Nowruz/pages/orgSetting/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react';
+import { HorizontalTabs } from 'src/Nowruz/modules/general/components/horizontalTabs';
+import { SearchDropdown } from 'src/Nowruz/modules/general/components/SearchDropdown';
+import Team from 'src/Nowruz/modules/orgSettings/components/team/';
+import Account from 'src/Nowruz/modules/settings/components/account/';
+
+export const Setting = () => {
+  const tabs = [
+    {
+      label: 'Account',
+      content: <Account />,
+      default: true,
+    },
+    {
+      label: 'Team',
+      content: <Team />,
+    },
+  ];
+  const items: any[] = [
+    { label: 'Account', value: 'Account' },
+    { label: 'Team', value: 'Team' },
+  ];
+
+  const [content, setContent] = useState<ReactNode>();
+
+  const setValue = (value) => {
+    console.log('value', value);
```

Comment:
remove `console.log`

Reply:
@satanmourner, orgSetting is not used in the project, I have removed it and developed setting for both user and org in one page. You can ignore this page, it will be removed after merge